Makefile

```diff
@@ -127,11 +161,41 @@ preload-images:
 		import_image quay.io/submariner/$${image}; \
 	done
 
+# Generate kustomization.yaml for bundle
+kustomization: manifests
+	operator-sdk generate kustomize manifests -q && \
+	cd config/manager && kustomize edit set image controller=$(IMG) && cd ../../ && \
+	(cd config/bundle && \
+	cat kustomization.template.yaml > kustomization.yaml && \
+	sed -i -e 's/$${VERSION}/$(VERSION)/g' kustomization.yaml && \
+	kustomize edit add annotation createdAt:"$(shell date "+%Y-%m-%d %T")" -f && \
+	cd ../../ )
+
+# Generate bundle manifests and metadata, then validate generated files.
+bundle: kustomization
```

Comment:
Maybe we should think about making CI cover this somehow?

Reply:
@dfarrell07 it fails because the calculated version includes some invalid characters for the bundle
you should specify the `VERSION` variable before running `make bundle`. kindly see the PR description for an example.

Reply:
Talking about this with @SteveMattar on the Automation WG sync today, the plan is to try to validate the version (if easy, ie doesn't have "-") and fail more quickly/clearly.

Imho the ideal is to have targets that pass by default (without any var overrides), but quickly and clearly failing is also okay.

Reply:
following our discussion I've added a validation using regex